aqt/installer.py

```diff
@@ -481,6 +481,10 @@ def _run_src_doc_examples(self, flavor, args, cmd_name: Optional[str] = None):
         else:
             qt_version = args.qt_version
             Cli._validate_version_str(qt_version)
+        # Override target/os for recent Qt
+        if Version(qt_version) >= Version("6.7.0"):
```

Comment:
This can be `if Version(qt_version) in SimpleSpec(">=6.7.0"):`

Reply:
or `if match(">=6.7.0", qt_version):` 

https://python-semanticversion.readthedocs.io/en/latest/reference.html#semantic_version.match

We use `version-in-spec` syntax in metadata.py, so it seems better to use similar one.